### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by horymury**
@saghul @hristoterezov 

Added it as a WIP as no comments are added

It can be used on Power Monitor and the rest as well, but in this PR it's only used in AOT

### CODE REVIEW

**Comment by horymury** [comment]
@hristoterezov  @saghul   switched to using the jitsi meet logger

**Comment by horymury** [comment]
@saghul  @hristoterezov   kind reminder :)

File: alwaysontop/main.js [hunk 1]
```diff
@@ -241,15 +261,39 @@ function setAspectRatioToResizeableWindow(win, aspectRatio) {
...
  * Initializes the always on top functionality in the main electron process.
  *
  * @param {BrowserWindow} jitsiMeetWindow - the BrowserWindow object which
+ * @param {Logger} isLoggingEnabled - whether logging should be enabled for this instance
+ * @param {Logger} loggerTransports - external loggers
  * displays Jitsi Meet
  */
-module.exports = function setupAlwaysOnTopMain(jitsiMeetWindow) {
+module.exports = function setupAlwaysOnTopMain(jitsiMeetWindow, loggerTransports) {
+    logger = require('jitsi-meet-logger').getLogger('AOT', loggerTransports || []);
```
Comment (hristoterezov) [inline alwaysontop/main.js:296]: Why do you need to require every time the function `setupAlwaysOnTopMain`?

**Comment by saghul** [comment]
Is this still alive?

**Comment by horymury** [comment]
@saghul  yep,I'll address the suggestions from @hristoterezov  soon.

**Comment by horymury** [comment]
@hristoterezov  I've addressed the review.

**Review by hristoterezov — APPROVED** [review]

**Pull request merged**